tests/Resources/ti.ui.view.addontest.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2015-Present by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+
+describe('Titanium.UI.View', function () {
+	let win;
+
+	this.timeout(5000);
+
+	afterEach((done) => {
+		if (win) {
+			let t = setTimeout(function () {
+				if (win) {
+					win = null;
+					done();
+				}
+			}, 3000);
+			win.addEventListener('close', function listener () {
+				clearTimeout(t);
+				if (win) {
+					win.removeEventListener('close', listener);
+				}
+				win = null;
+				done();
+			});
+			win.close();
+		} else {
+			win = null;
+			done();
+		}
+	});
+
+	it.android('touchFeedback', (finish) => {
```

Comment:
I appreciate the effort in creating a test, but I don't think it's really verifying anything since this is an interactive UI change, correct?

Reply:
Right. It doesn't verify anything, but it does exercise the native API to make sure it doesn't crash. So, I guess it's better than nothing? :man_shrugging:

Ideally, QE should test this by hand via a functional/smoke test since this is a visual feature.